src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -468,6 +468,50 @@
                 </div>
               </div>
 
+              <!-- Lifecycle -->
+              <div *ngIf="editing"
+                   class="form-group row">
+              <label i18n
+                     class="cd-col-form-label"
+                     for="id">Lifecycle
+                <cd-helper>JSON or XML formatted document</cd-helper>
```

Comment:
I personally though this was non essential information and since there also is the validator error and text message, I left it as a helper.

But If you would prefer it to have it upfront we can change it. I'm not sure where should lay this text, where would you place the text @cloudbehl? So I can modify it accordingly